app/src/main/java/io/appium/uiautomator2/model/internal/CustomUiDevice.java

```diff
@@ -86,6 +89,15 @@ public Instrumentation getInstrumentation() {
         return mInstrumentation;
     }
 
+    public UiAutomation getUiAutomation() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+            int flags = Configurator.getInstance().getUiAutomationFlags();
+            return getInstrumentation().getUiAutomation(flags);
+        } else {
```

Comment:
this else is redundant

Reply:
removed else https://github.com/appium/appium-uiautomator2-server/pull/463/commits/4be80049b1d5c8ebbe2959b4068273eccef725a1